### PR TITLE
added sacura tests to keda test job

### LINKED ISSUE #3711: Add Sacura Tests to KEDA test job
**Problem**
Currently we are not running the Sacura test in the KEDA test job. To catch performance issues, we should enable them.

**[Persona:](https://github.com/knative/eventing/blob/master/docs/personas.md)** 
Which persona is this feature for?

**Exit Criteria**
Sacura tests run in the KEDA job

**Time Estimate (optional):** 
How many developer-days do you think this may take to resolve? 1

**Additional context (optional)**
Add any other context about the feature request here.

### PR DESCRIPTION
**Pull request by NeerajNagure**
Fixes #3711 

## Proposed Changes
added a script in keda-reconciler-tests.sh that will run sacura tests in it

:gift: Added sacura tests in keda job

### CODE REVIEW

File: test/keda-reconciler-tests.sh [hunk 1]
```diff
@@ -41,4 +41,6 @@ header "Running tests"
 
 go_test_e2e -timeout=1h -run=KafkaSource ./test/e2e_new/... || fail_test "E2E (new) suite failed"
 
+go_test_e2e -tags=sacura -timeout=40m ./test/e2e/... || fail_test "E2E sacura tests failed"
```
Comment (Cali0707) [inline test/keda-reconciler-tests.sh:44]: Can you verify that this is not in LOCAL_DEVELOPMENT before running these tests? We should probably also still have the `-run=KafkaSource` flag set for these tests.

To check the LOCAL_DEVELOPMENT, see https://github.com/knative-extensions/eventing-kafka-broker/blob/315e260db7a27c0e899d6b2a34eacc8e5a06abf3/test/reconciler-tests.sh#L73-L75

**Review by Cali0707 — CHANGES_REQUESTED** [review]
Thanks for starting on this @NeerajNagure ! Just two small changes

**Comment by NeerajNagure** [comment]
@Cali0707 I have added the -run=kafkasource flag and also checked whether the environment is LOCAL_DEVELOPMENT or not before running sacura tests

File: test/keda-reconciler-tests.sh [hunk 2]
```diff
@@ -41,4 +41,8 @@ header "Running tests"
 
 go_test_e2e -timeout=1h -run=KafkaSource ./test/e2e_new/... || fail_test "E2E (new) suite failed"
 
+if ! ${LOCAL_DEVELOPMENT}; then
+go_test_e2e -run=KafkaSource -tags=sacura -timeout=40m ./test/e2e/... || fail_test "E2E sacura tests failed"
```
Comment (pierDipi) [inline test/keda-reconciler-tests.sh:45]:
```suggestion
	go_test_e2e -run=KafkaSource -tags=sacura -timeout=40m ./test/e2e/... || fail_test "E2E sacura tests failed"
```

**Review by Cali0707 — APPROVED** [review]
/lgtm
/approve

Thanks @NeerajNagure !!

**Pull request merged**